java/test/jmri/jmrit/display/layoutEditor/LayoutEditorToolsTest.java

```diff
@@ -21,17 +21,25 @@ public void testCtor() {
         LayoutEditor e = new LayoutEditor();
         LayoutEditorTools t = new LayoutEditorTools(e);
         Assert.assertNotNull("exists", t);
-        JUnitUtil.dispose(e);
+        e.dispose();
```

Comment:
This change needs to be reverted as well.

Reply:
Done!